### ISSUE TITLE
mdvsex-deck

### ISSUE THREAD

**Issue by janosh** [pngwn/MDsveX#501]
Has anyone looked into making slides with `mdsvex`? I'm wondering if there's an equivalent to React's [mdx-deck](https://github.com/jxnblk/mdx-deck) for Svelte.

**Comment by riziles** [comment]
Yes! Check this out: https://github.com/bennymi/presveltation

**Comment by janosh** [comment]
Thanks for sharing. Have you used it? Markdown support doesn't look like a 1st party feature. I'm wondering how much Svelte the average slide requires.

**Comment by riziles** [comment]
Yes. It's pretty easy to use. I'm sure @bennymi would welcome a PR if you have any ideas on how to improve. If you want to make a deck, I'd recommend just cloning the repo. It's a little faster than adding it to a project. I've been meaning to create a starter template, but haven't had time.

**Comment by bennymi** [comment]
Hi @janosh, @riziles got `mdsvex` working in the slides and it's fairly straight forward. The set up of the library still needs to be simplified a bit though, so just cloning the repo is definitely a good choice for now. I'll update it in the future when I'm finished with my exams, but PRs are also always welcome if you have improvement ideas.

**Comment by riziles** [comment]
@janosh and @bennymi , I created a branch where I stripped out the docs folder and set up the root path to reset the slide position. You can run 

```bash
degit riziles/presveltation_fork#template
```

I'm sure there's more I can do to clean it up, but it's functional.
